canonical-web-and-design/jaas-dashboard
# Add empty state for model relations view

## Done

Add empty state for model relations view

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Click on a model that has no relations to go to detail view
- Click the "Relations" tab 
- See the following empty state message appears

<img width="1505" alt="Screenshot 2020-11-24 at 11 33 50" src="https://user-images.githubusercontent.com/505570/100093486-7ed34080-2e4f-11eb-883e-fd1ac702dd31.png">

## Details

Fixes #750
